bootstrap/helpers/proxy.php

```
@@ -334,3 +334,94 @@ function generateDefaultProxyConfiguration(Server $server, array $custom_command
+        return null;
+    } catch (\Exception $e) {
+        Log::error("getExactTraefikVersionFromContainer: Server '{$server->name}' (ID: {$server->id}) - Error: ".$e->getMessage());
+
+        return null;
+    }
+}
+
+function getTraefikVersionFromDockerCompose(Server $server): ?string
+{
+    try {
+        Log::debug("getTraefikVersionFromDockerCompose: Server '{$server->name}' (ID: {$server->id}) - Starting version detection");
```

PHPMD (2.15.0)

Avoid using static access to class 'Log' in method 'getTraefikVersionFromDockerCompose'. (undefined)

(StaticAccess)